linux/src/threadpool_linux.c

```diff
@@ -503,7 +503,7 @@ static void threadpool_timer_dispose(THREADPOOL_TIMER * timer)
     // the callback to use it if we hit this window.
 
     /* Codes_SRS_THREADPOOL_LINUX_07_096: [ threadpool_timer_dispose shall call ThreadAPI_Sleep to allow timer resources to clean up. ]*/
-    ThreadAPI_Sleep(30);
+    ThreadAPI_Sleep(35);
```

Comment:
>35 [](http://example.com/codeflow?start=20&length=2)

I find it very interesting that we had to change from 30 to 35. Did some test fail?

Reply:
Then why not 40?
Or 45?
Or ...